### ISSUE TITLE
Tooltip position is wrong when using keyboard to access and open 

### ISSUE THREAD

**Issue by jingx-xue** [carbon-design-system/carbon#3682]
## Environment

> Operating system: WIN 10

> Browser: latest chrome 

> Automated testing tool and ruleset: NA

> Assistive technology used to verify: NA

## Detailed description

> What version of the Carbon Design System are you using?
 10.4.0

> What did you expect to happen?
the position of the tooltip content should be in right below of the text
> What happened instead?
the position of the tooltip content is jump to the far right 
> What WCAG 2.1 checkpoint does the issue violate?

## Steps to reproduce the issue

1. resize the browser 
2. use keyboard to open the tooltip 

> Please create a reduced test case in CodeSandbox
>
I test with http://react.carbondesignsystem.com/?path=/story/tooltip--no-icon
also with https://codesandbox.io/s/codesandbox-hojfx
if you minimized the screen and scroll down the page, the position become wrong,

## Additional information

- Screenshots or code
- Notes

**Comment by jingx-xue** [comment]
![image](https://user-images.githubusercontent.com/48215968/62649696-4d192c00-b923-11e9-89be-7df4f44149fe.png)
